cypress-tests/cypress/e2e/PaymentUtils/Stripe.js

```diff
@@ -109,6 +130,24 @@ export const connectorDetails = {
         status: 200,
         body: {
           status: "requires_customer_action",
+          setup_future_usage: "on_session",
+          payment_method_data: {
+            card: {
+              last4: "3155",
+              card_type: "CREDIT",
+              card_network: "Visa",
+              card_issuer: "INTL HDQTRS-CENTER OWNED",
+              card_issuing_country: "UNITEDSTATES",
+              card_isin: "400000",
+              card_extended_bin: null,
+              card_exp_month: "10",
+              card_exp_year: "25",
+              card_holder_name: null,
+              payment_checks: null,
+              authentication_data: null,
+            },
+            billing: null,
+          },
```

Comment:
you can make this a variable above and use it everywhere in configs. something like:

```suggestion
          payment_method_data: pmd,
```

define `pmd` in the beginning of the file or in `commons` something like:

```js
pmd = {
  card: {
    last4: "3155",
    card_type: "CREDIT",
    card_network: "Visa",
    card_issuer: "INTL HDQTRS-CENTER OWNED",
    card_issuing_country: "UNITEDSTATES",
    card_isin: "400000",
    card_extended_bin: null,
    card_exp_month: "10",
    card_exp_year: "25",
    card_holder_name: null,
    payment_checks: null,
    authentication_data: null,
  },
  billing: null,
},
```